webapp/channels/src/components/post/user_profile.tsx

```diff
@@ -95,7 +95,11 @@ const PostUserProfile = (props: Props): JSX.Element | null => {
                 />
             );
 
-            botIndicator = (<BotTag/>);
+            // user profile component checks and add bot tag in case webhook is from bot account, but if webhook is from user account we need this.
```

Comment:
Can we add more/better description of why we're doing this? Maybe mention the jira ticket in code comment itself? Without the context of the community thread, the code looks weird - we add bot tag if it's not a bot.